tests/pamiq_core/console/test_cui.py

```diff
@@ -0,0 +1,212 @@
+            else:
+                assert f'Command "{command_name}" not executed.' not in captured.out
+
+    def test_help_command(
+        self, console: Console, capsys: pytest.CaptureFixture[str]
+    ) -> None:
+        console.do_help("")
+        captured = capsys.readouterr()
+        # Check if "help" explains all commands.
+        captured_commands: list[str] = []
+        for line in captured.out.split("\n"):
+            # catch the format "cmd1/cmd2/... Explain"
+            match = re.compile(r"^([\w/]+)").match(line)
+            if match:
+                cmds = match.group(1).split("/")
+                captured_commands += cmds
+        assert set(console.get_all_commands()) == set(captured_commands)
+
+    def test_do_pause(
+        self,
+        console: Console,
+        mock_httpx: MagicMock,
+        capsys: pytest.CaptureFixture[str],
+    ) -> None:
+        mock_httpx.post.return_value.text = json.dumps({"result": "test do_pause"})
+        console.do_pause("")
+        mock_httpx.post.assert_called_once_with("http://localhost:8391/api/pause")
+        captured = capsys.readouterr()
+        assert "test do_pause" in captured.out
+
+    def test_do_p_as_alias(self, console: Console, mock_httpx: MagicMock) -> None:
+        with patch.object(console, "do_pause") as mock_pause:
+            console.do_p("")
+            mock_pause.assert_called_once_with("")
+
+    def test_do_resume(
+        self,
+        console: Console,
+        mock_httpx: MagicMock,
+        capsys: pytest.CaptureFixture[str],
+    ) -> None:
+        mock_httpx.post.return_value.text = json.dumps({"result": "test do_resume"})
+        console.do_resume("")
+        mock_httpx.post.assert_called_once_with("http://localhost:8391/api/resume")
+        captured = capsys.readouterr()
+        assert "test do_resume" in captured.out
+
+    def test_do_r_as_alias(self, console: Console, mock_httpx: MagicMock) -> None:
+        with patch.object(console, "do_resume") as mock_resume:
+            console.do_r("")
+            mock_resume.assert_called_once_with("")
+
+    @pytest.mark.parametrize("users_answer", ["y", "yes"])
+    def test_do_shutdown_yes(
+        self,
+        console: Console,
+        mock_httpx: MagicMock,
+        capsys: pytest.CaptureFixture[str],
+        monkeypatch: pytest.MonkeyPatch,
+        users_answer: str,
+    ) -> None:
+        monkeypatch.setattr(
+            "builtins.input", lambda prompt: users_answer
+        )  # Mock input() to return <users_answer>.
+        mock_httpx.post.return_value.text = json.dumps({"result": "test do_shutdown"})
+        result = console.do_shutdown("")
+        mock_httpx.post.assert_called_once_with("http://localhost:8391/api/shutdown")
+        captured = capsys.readouterr()
+        assert "test do_shutdown" in captured.out
+        assert result is True
+
+    @pytest.mark.parametrize("users_answer", ["other_strings"])
```

Comment:
デフォルトの`n`, `N`も追加してほしい
```suggestion
    @pytest.mark.parametrize("users_answer", ["n", "N", "other_strings"])
```